sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -120,7 +120,9 @@ object JDBCRDD extends Logging {
       case StringStartsWith(attr, value) => s"${attr} LIKE '${value}%'"
       case StringEndsWith(attr, value) => s"${attr} LIKE '%${value}'"
       case StringContains(attr, value) => s"${attr} LIKE '%${value}%'"
-      case In(attr, value) => s"$attr IN (${compileValue(value)})"
+      case In(null, value) => "NULL"
```

Comment:
Yeah. `NULL` in predicates means `UNKNOWN`. 

FYI, 
- our source code for the `IN` operator: https://github.com/apache/spark/blob/84284e8c82542d80dad94e458a0c0210bf803db3/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala#L141-L142

Oracle Docs of `NULL`: https://docs.oracle.com/cd/B19306_01/server.102/b14200/sql_elements005.htm